components/data/data-services/org.wso2.micro.integrator.dataservices.core/src/main/java/org/wso2/micro/integrator/dataservices/core/DataServiceFactory.java

```diff
@@ -186,6 +186,16 @@ public static DataService createDataService(OMElement dbsElement,
                                                                        itr.next()));
             }
 
+            String swaggerLocation = dbsElement.getAttributeValue(new QName(DBSFields.SWAGGER_LOCATION));
+
+            if (swaggerLocation != null && !swaggerLocation.isEmpty()) {
```

Comment:
Can we use org.apache.commons.lang.StringUtils.isNotEmpty(swaggerResourcePath) for this?

Reply:
+1 fixed